templates/zerver/api/get-stream-topics.md

```diff
@@ -0,0 +1,70 @@
+# Get topics in a stream
+
+Get all the topics in a specific stream
+
+`GET {{ api_url }}/v1/users/me/<stream_id>/topics`
+
```

Comment:
There needs to be an **Arguments** section that documents all the potential arguments that can be passed to this API endpoint. Use the `generate_api_arguments_table` Markdown extension and `templates/zerver/api/arguments.json` to document the arguments. :)

Reply:
I just didn't add that because this endpoint in particular doesn't require any arguments (besides the `<stream_id>` which is included in the URL). Should I add it anyways?

Reply:
@YagoGG: Yeah I think we should add it. It is a good thing we can be consistent across all of our API docs and document all arguments as exhaustively as possible. :)